okhttp/src/main/java/okhttp3/internal/ws/CompressionCorrector.kt

```diff
@@ -0,0 +1,49 @@
+package okhttp3.internal.ws
+
+import okio.Buffer
+import okio.ByteString.Companion.decodeHex
+
+/**
+ * Applies post-deflate and pre-inflate buffer transformations
+ * described in [rfc7692#section-7.2](https://tools.ietf.org/html/rfc7692#section-7.2)
+ */
+object CompressionCorrector {
+
+  private val EMPTY_DEFLATE_BLOCK = "000000ffff".decodeHex()
+  private const val LAST_OCTETS_TO_REMOVE_AFTER_DEFLATION = 4
```

Comment:
`LAST_OCTETS_COUNT_TO_REMOVE_AFTER_DEFLATION`?

Damn, I’m practically hearing how this name screams at me with these caps letters...

Reply:
Done